features/attached_commands.feature

```diff
@@ -751,8 +748,8 @@ Feature: Command behaviour when attached to an Ubuntu Pro subscription
         { "metering_timer": "notanumber", "update_messaging_timer": -10 }
         """
         And I run `python3 /usr/lib/ubuntu-advantage/timer.py` with sudo
-        Then I verify that running `grep "Invalid value for update_messaging interval found in config." /var/log/ubuntu-advantage-timer.log` `with sudo` exits `0`
-        And I verify that the timer interval for `update_messaging` is `21600`
+        Then I verify that running `sh -c journalctl -t python3 | grep "Invalid value for update_messaging interval found in config."` `with sudo` exits `0`
```

Comment:
If you want to select the timer logs I suggest using the `-u` option for journalctl
```suggestion
        Then I verify that running `sh -c journalctl -u ua-timer.service | grep "Invalid value for update_messaging interval found in config."` `with sudo` exits `0`
```